src/common/DisabledButtonInfo.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { ReactNode, useEffect } from 'react'
+import Tooltip, { TooltipProps } from '@mui/material/Tooltip'
+
+function DisabledButtonInfo({
+  conditions,
+  placement,
+  children,
+}: {
+  conditions: string[]
+  placement?: TooltipProps['placement']
+  children: ReactNode
+}) {
+  const [titleText, setTitleText] = React.useState<string>('')
+
+  useEffect(() => {
+    if (conditions !== undefined) {
+      let updatedTitleText = ''
+      conditions.forEach((condition) => {
+        if (condition !== '') {
+          updatedTitleText = `${updatedTitleText} ${condition}`
+        }
+      })
+      setTitleText(updatedTitleText)
+    }
+  }, [conditions])
+
+  return (
+    <Tooltip title={titleText} arrow placement={placement !== undefined ? placement : 'right'}>
```

Comment:
Another convention I've found useful is to always provide true boolean props first, so in this case:
```
<Tooltip arrow title={titleText} placement={placement || 'right'}>
```
Again, not something worth changing unless it's a convention we want to enforce.